pydis_site/apps/api/models/bot/infraction.py

```diff
@@ -15,7 +15,8 @@ class Infraction(ModelReprMixin, models.Model):
         ("mute", "Mute"),
         ("kick", "Kick"),
         ("ban", "Ban"),
-        ("superstar", "Superstar")
+        ("superstar", "Superstar"),
+        ("voice_ban", "Voice Ban")
```

Comment:
```suggestion
        ("voice_ban", "Voice Ban"),
```

I know it wasn't there before, but future cases could be avoiding the same two-line diffs with it.